lib/pages/chat/events/event_video_player.dart

```diff
@@ -74,7 +77,13 @@ class EventVideoPlayer extends StatelessWidget {
         color: Colors.black,
         child: InkWell(
           mouseCursor: SystemMouseCursors.click,
-          onTap: () => _onTapVideo(context),
+          onTap: () {
+            if (onVideoTapped != null) {
+              onVideoTapped!.call();
+            } else {
+              _onTapVideo(context);
+            }
+          },
```

Comment:
what is the different: `onVideoTapped` vs `_onTapVideo`?

Reply:
its the same, so i remove the below, and use only in the case of `ChatDetailsPage`